docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+version: "3.9"
+
+services:
+  todo_app:
+    container_name: todo_app
+    image: todo-app:dev
```

Comment:
Could you see how to make this compose file use the `Dockerfile` directly?

Reply:
I could use the  `dockerfile: Dockerfile` attr under `build`?